admin-tool/src/aio/configure.rs

```diff
@@ -88,6 +94,9 @@ impl Default for Configuration {
             manufacturing_disable_key_storage_tpm: true,
             manufacturing_use_secp256r1: false,
 
+            ov_registration_period: Some(600),
+            ov_re_registration_window: Some(100),
```

Comment:
I wonder whether we should define a new fdo-constants or fdo-defaults for that since it doesn't look right to me to put those constants in the fdo-data-formats crate. what do you think?

Reply:
I think the best fit would be `util/src/servers/configuration/owner_onboarding_server.rs`. Not sure if putting constants there is an antipattern in rust or goes against something though.